### PR TITLE
9 Create Atom parser

### PR DESCRIPTION
**Pull request by PDendys**
This closes issue #9 

# Overwiev
Geting atom feed from given url

# Testing scenarion
- Implement AtomParser.parsenews() in MainWindow.xaml.cs
- [x] You're able to get list of news objects 

# Note
example:
var feed = AtomParser.parsenews("https://www.theregister.co.uk/data_centre/headlines.atom");

### CODE REVIEW

File: GPP-1/AtomParser.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            Atom10FeedFormatter formatter = new Atom10FeedFormatter();
+
+            using (XmlReader reader = XmlReader.Create(url))
+            {
+                formatter.ReadFrom(reader);
+            }
+
+            foreach (SyndicationItem item in formatter.Feed.Items)
+            {
+                _newsList.Add(NewsFactory.CreateNews("Category", Regex.Replace(item.Summary.Text, "<.*?>", string.Empty), item.Links[0].Uri.AbsoluteUri, item.Title.Text, item.LastUpdatedTime.Date));
```
Comment (bartoszboruta) [inline GPP-1/AtomParser.cs]: Walnij to jedno pod drugim i powywalaj entery

File: GPP-1/AtomParser.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        public static List<News> parsenews(string url)
+        {
+            List<News> _newsList = NewsFactory.CreateEmptyList();
+
+            Atom10FeedFormatter formatter = new Atom10FeedFormatter();
+
+            using (XmlReader reader = XmlReader.Create(url))
+            {
+                formatter.ReadFrom(reader);
+            }
```
Comment (mateuszwu) [inline GPP-1/AtomParser.cs]: consider use:
- `formatter.ReadFrom(XmlReader.Create(url));`
or
- `var content = XmlReader.Create(url);
formatter.ReadFrom(content);`

File: GPP-1/AtomParser.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+            foreach (SyndicationItem item in formatter.Feed.Items)
+            {
+                _newsList.Add(NewsFactory.CreateNews("Category", Regex.Replace(item.Summary.Text, "<.*?>", string.Empty), item.Links[0].Uri.AbsoluteUri, item.Title.Text, item.LastUpdatedTime.Date));
+
+            }
+
+            return _newsList;
+
```
Comment (mateuszwu) [inline GPP-1/AtomParser.cs]: remove empty line

File: GPP-1/AtomParser.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
```
Comment (mateuszwu) [inline GPP-1/AtomParser.cs]: same here

File: GPP-1/AtomParser.cs [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.XPath;
+using System.Text.RegularExpressions;
+
+namespace GPP_1
+{
+    static class AtomParser
+    {
+
```
Comment (mateuszwu) [inline GPP-1/AtomParser.cs]: remove empty line

**Pull request merged**